cfep-xx.md

```diff
@@ -0,0 +1,80 @@
+
+<table>
+<tr><td> Title </td><td> Make all Rust packages auditable </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Uwe L. Korn &lt;uwelk@xhochy.com&gt; </td></tr>
+<tr><td> Created </td><td> Jun 23, 2025</td></tr>
+<tr><td> Updated </td><td> Jun 23, 2025</td></tr>
+<tr><td> Discussion </td><td> N/A </td></tr>
+<tr><td> Implementation </td><td> <a href="https://github.com/conda-forge/rust-activation-feedstock/pull/79">rust-activation-feedstock#79</a> </td></tr>
+</table>
+
+## Abstract
+
+This proposal suggests integrating [`cargo‑auditable`](https://github.com/rust-secure-code/cargo-auditable) into the build process for all Rust-based feedstocks on conda‑forge.
+By enabling dependency‑tree embedding within compiled binaries, this integration would enhance downstream vulnerability auditing and transparency.
+
+## Motivation
+
+`cargo-auditable` embeds dependency information as a JSON blob into a dedicated linker section with [minimal overhead (\~4 kB), even for large dependency graphs (>400 crates)](https://github.com/rust-secure-code/cargo-auditable?tab=readme-ov-file#doesnt-this-bloat-my-binary).
+This enables downstream tooling (e.g. `cargo audit bin`, Trivy, Syft) to perform highly accurate vulnerability scans.
+
+This brings the Rust binaries on conda-forge en par with the Go binaries that have this enabled by default (see the output of `go version -m <binary>`).
+Multiple distributions (Alpine, NixOS, openSUSE, Void, Chimera) already adopt `cargo-auditable` in Rust package builds (see [Adoption](https://github.com/rust-secure-code/cargo-auditable?tab=readme-ov-file#adoption))
+
+## Proposal
```

Comment:
I'd recommend adding a Specification section where you are less specific here when it comes to implementation details, and instead focus on the goal to achieve. The concrete details about the suggested implementation can be done in a "Suggested implementation" section if needed, but not in the Specification.

For example, instead of saying "Add `cargo-auditable` as a hard dependency in `rust-activation`", you could say "Ensure `cargo-auditable` is installed by default in build environments that need `rust`". This way the specification (which is the thing we would be actually voting) wouldn't get outdated if the implementation details change over time.